src/System.Devices.Gpio/System.Devices.Gpio/GpioDriver.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System.IO;
```

Comment:
Missing header

// Copyright (c) Microsoft. All rights reserved.
// Licensed under the MIT license. See LICENSE file in the project root for full license information.